draft-ietf-tls-tls13.md

```diff
@@ -1937,12 +1943,24 @@ random
   This structure is generated by the server and MUST be
   generated independently of the ClientHello.random.
 
+legacy_session_id_echo
+: The contents of the client's legacy_session_id field. Note that
+  this field is echoed even if the client's value corresponded to
+  a cached pre-TLS 1.3 session which the server has chosen not
+  to resume. A client which receives a legacy_session_id field
+  which does not match what it sent in the ClientHello
+  MUST abort the handshake with an "illegal_parameter"
+  alert.
+
 cipher_suite
 : The single cipher suite selected by the server from the list in
   ClientHello.cipher_suites. A client which receives a cipher suite
   that was not offered MUST abort the handshake with an "illegal_parameter"
   alert.
 
+legacy_compression_method
+: A single byte of value 0.
```

Comment:
What should a client do if it receives something else?

Reply:
`illegal_parameter` seems like a correct choice

Reply:
Yes, but we don't need to say it here because illegal_parameter's definitions says o.